packages/terra-table/docs/Table.md

```diff
@@ -0,0 +1,40 @@
+# Terra Table
+
+The Terra Table is a structural component to arrange content within a table.
```

Comment:
It's realy picky, but this almost sounds like it could be used for layouts. I think it could be better to say that Terra Table is used for building standard html tables.

Reply:
Updated.